src/main/java/uk/gov/pay/ledger/common/TransactionState.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.pay.ledger.common;
```

Comment:
think this should go in transaction package unless there is a good reason not to